### PR TITLE
Fix bug about genesis file struct

### PR DESCRIPTION
**Pull request by mask-pp**

### CODE REVIEW

**Comment by ChuhanJin** [comment]
Can one of the admins verify this patch?

**Review by 0xmountaintop — APPROVED** [review]

**Review by icemelon — COMMENTED** [review]
Could you add a test to capture the error?

**Comment by mask-pp** [comment]
> Could you add a test to capture the error?



> Could you add a test to capture the error?

I will try to add it, although it's a little hassle

File: Makefile [hunk 1]
```diff
@@ -29,6 +29,9 @@ ios:
 	@echo "Import \"$(GOBIN)/Geth.framework\" to use the library."
 
 test: all
+	# genesis test
+	cd ${PWD}/cmd/geth; go test -test.run TestCustomGenesis
+	# moudle test
```
Comment (icemelon) [inline Makefile]:
```suggestion
	# module test
```

File: internal/cmdtest/test_cmd.go [hunk 2]
```diff
@@ -165,7 +167,7 @@ func (tt *TestCmd) ExpectRegexp(regex string) (*regexp.Regexp, []string) {
 	)
 	tt.withKillTimeout(func() { matches = re.FindReaderSubmatchIndex(rtee) })
 	output := rtee.buf.Bytes()
-	if matches == nil {
+	if matches == nil && !bigCmp(string(output), regex) {
```
Comment (icemelon) [inline internal/cmdtest/test_cmd.go:170]: why do we need to modify this?

Comment (mask-pp) [reply internal/cmdtest/test_cmd.go:170]: In order to compare `9.04625697166532776746648320380374280103671755200316906558262375061821325312e+74` with `0x200000000000000000000000000000000000000000000000000000000000000`.

**Review by icemelon — APPROVED** [review]
lgtm

**Pull request merged**